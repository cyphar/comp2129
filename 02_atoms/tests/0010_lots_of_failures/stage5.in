QUIT lol
QUIT   a  a 2-2
START 4 2 2
QUIT 12 3
QUIT   aaa   
QUIT
