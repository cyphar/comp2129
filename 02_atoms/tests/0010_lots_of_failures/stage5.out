Invalid Command

Invalid Command

Game Ready
Red's Turn

Invalid Command

Invalid Command

Bye!
